Processen/CR-PR-API-Beheer.md

```diff
@@ -14,6 +14,32 @@ Het onderstaande diagram beschrijft hoe er wordt om gegaan met issues die in het
 
```

Comment:
Het prioriteren en ready maken van user stories is onderdeel van scrum. Nu staat er een blokje scrum nr. 2.26. Maar genoemde activiteiten zijn daar al onderdeel van. Dus bijna de gehele onderkant van het schema is het Scrum proces. Diagram kan veel eenvoudiger worden daardoor.

Belangrijk is ook dat er wsch. niet altijd een sprint hoeft te worden gedraaid.Dat bekijken we aan de hand van de user stories en bugs die we hebben staan. De eerste maanden zal dat zeker nodig zijn, later misschien niet altijd.

Reply:
Ik schets even mijn gedachtegang. 

Ik heb in 2.25 de term 'Release backlog' geïntroduceerd. Die bevat alle (wens-)issues waarvan jij als PO en alle (bug-)issues waarvan wij als beheerders vinden dat die moeten worden meegenomen in de komende release. Een mechanisme dus om vooraf de scope van een scrum project te kunnen bepalen. 

De issues in de Release backlog worden in 1 of meer sprints afgehandeld dus in het scrum proces (2.26) wordt VOOR elke sprint bepaald welke issues we die sprint gaan oppakken. Dus voorafgaand aan 2.26 is er nog geen sprake van scrum. En natuurlijk starten we alleen een scrum traject als er voldoende issues in de Release backlog staan.

Ik kan me voorstellen dat je vindt dat het ready maken in het scrum proces plaats moet vinden. In principe zou ik dat al kunnen realiseren door de tekst van 2.5 te wijzigen in 'Issue helder maken'. Dat betekent echter dat de ontwikkelaar in 2.8 met een issue aan de slag moet dat niet ready is. De vraag is of dat erg is.

Reply:
Het prioriteren en op de de sprint/release backlog plaatsen van issues is gewoon onderdeel van Scrum. Dat sprint planning namelijk. Nogmaals je moet niet proberen het sprint proces opnieuw te beschrijven in een workflow. Geldt ook voor refinement, review, retrospective.

Reply:
Zoals je ziet heb ik het proces sterk vereenvoudigd en de onderdelen die al in scrum zitten verwijderd.
Wat betreft de terugkoppeling heb ik e.e.a. wat neutraler verwoord. Vooralsnog vinden wij echter nog steeds dat er een actieve terugkoppeling plaats moet vinden.